### PR TITLE
Pynfs acl

### PR DESCRIPTION
**Pull request by aloganat**

### CODE REVIEW

File: common-scripts/basic-gluster.sh [hunk 1]
```diff
@@ -128,3 +128,12 @@ then
 	echo "Export ${GLUSTER_VOLUME} is not available"
 	exit 1
 fi
+
+if [ "${ENABLE_ACL}" == "True" ]
+then
+  conf_file="/etc/ganesha/exports/export."${GLUSTER_VOLUME}".conf"
+  sed -i s/'Disable_ACL = .*'/'Disable_ACL = false;'/g ${conf_file}
+  cat ${conf_file}
+  dbus-send --type=method_call --print-reply --system  --dest=org.ganesha.nfsd /org/ganesha/nfsd/ExportMgr  org.ganesha.nfsd.exportmgr.UpdateExport string:${conf_file} string:"EXPORT(Export_Id = 2)"
```
Comment (soumyakoduri) [inline common-scripts/basic-gluster.sh]: Though the ExportID is always '2' for now, can we grep it from .conf file? In case if it changes in future.

Comment (aloganat) [reply common-scripts/basic-gluster.sh]: Comments incorporated.

File: common-scripts/basic-gluster.sh [hunk 2]
```diff
@@ -128,3 +128,17 @@ then
...
+
+#Enabling ACL for the volume if ENABLE_ACL param is set to True
+if [ "${ENABLE_ACL}" == "True" ]
+then
+  conf_file="/etc/ganesha/exports/export."${GLUSTER_VOLUME}".conf"
+  sed -i s/'Disable_ACL = .*'/'Disable_ACL = false;'/g ${conf_file}
+  cat ${conf_file}
+
+  #Parsing export id from volume export conf file
+  export_id=$(awk '/Export_Id /{print}' ${conf_file} | cut -f2 -d "=" | xargs | sed 's/;//')
```
Comment (nixpanic) [inline common-scripts/basic-gluster.sh]: Why the `xargs` in there? I think that is not needed.

Comment (aloganat) [reply common-scripts/basic-gluster.sh]: I have used xargs to strip the whitespaces in the export_id string

**Review by nixpanic — APPROVED** [review]

**Pull request merged**